<launch>

  <arg name="debug" default="false"/>
  <arg name="source_data" default="interactive_marker"/>  <!-- interactive_marker or topic -->
  <param name="whole_body_kinematic_controler/source_data" value="$(arg source_data)" />
  
  <!-- Upload capsule collision operations  & description-->
  <include file="$(find tiago_wbc)/launch/tiago_capsule_description.launch" />

  <!-- Configure wbc -->
  <rosparam command="load"
    file="$(find tiago_wbc)/config/tiago_environment_collision_avoidance_wbc.yaml" />
  
  <!-- Spawn controller -->
  <node name="wbc_controllers_spawner"
        pkg="controller_manager" type="spawner" output="screen"
        args="whole_body_kinematic_controler" />
        
</launch>
